tests/functional/pages/confirmation.page.js

```diff
@@ -0,0 +1,22 @@
+class Confirmation {
```

Comment:
I think if we named this class `ConfirmationPage` it would make the places it's used in the tests more readable